packages/core/src/components/grid/grid.scss

```diff
@@ -1,7 +1,23 @@
 @import '~@atomium/scss-utils/index';
 
+:host {
+  --atom-grid-padding: 0;
```

Comment:
I was in doubt about the width, because we don't have the sizes defined and I don't know exactly what sizes we need to define. The padding, I understand, was abstracted, right?

<img width="705" alt="Captura de Tela 2024-05-27 às 11 41 47" src="https://github.com/juntossomosmais/atomium/assets/54173994/ee038648-2482-45ea-b402-bc3f6e774097">

Reply:
@felipefialho I think it's worth maybe opening a thread with UX.

Reply:
Make sense! Let's keep it that way